pkgs/tools/networking/nqptp/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, autoreconfHook
+, pkg-config
+}:
+
+with lib;
+stdenv.mkDerivation rec {
+  version = "1.1-dev-168-g3444047";
+  pname = "nqptp";
```

```suggestion
  pname = "nqptp";
  version = "unstable-XXXX-XX-XX";
```
We don't do such version numbers. Please fill in the iso date.